kousa/lib/kousa/user.ex

```diff
@@ -54,4 +54,19 @@ defmodule Kousa.User do
       _ -> {:error, "tried to ban #{user_id_to_ban} but that user didn't exist"}
     end
   end
+
+  def set_staff_and_contributions(username_to_change, staff, contributions, opts) do
+    authorized_github_id = Application.get_env(:kousa, :ben_github_id, "")
+
+    with %{githubId: ^authorized_github_id} <- Users.get_by_id(opts[:admin_id]) do
+      user_to_change = Users.get_by_username(username_to_change)
+      Users.set_staff(user_to_change.id, staff)
+      Users.set_contributions(user_to_change.id, contributions)
```

Comment:
these should be combined into a single update, see how non admin user update works

Reply:
done